static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
@brianparry  firewall and privilege control only cover security problem. southbound and northbound API splitting don't only cover security, but also have other considerations, please see my comments at https://rackhd.atlassian.net/browse/RAC-1447.  I also don't want to this part impact your progress, so I commented  in RAC-1447.